src/lib/analytics/AnalyticsClass.js

```diff
@@ -227,6 +230,8 @@ export class AnalyticsClass {
     }
 
     logger.debug('fired event', { event, data })
+    emitter.emit('fireEvent', { event, data })
+    emitter.emit(event, data)
```

Comment:
why? this is duplicate

Reply:
to be able to listen for any event we send to for some particular